qds_sdk/cluster.py

```diff
@@ -68,14 +72,26 @@ def list(cls, state=None):
             List of clusters satisfying the given criteria
         """
         conn = Qubole.agent()
+        params = {}
+        if page:
+            params['page'] = page
+        if per_page:
+            params['per_page'] = per_page
+        params = None if not params else params
+        cluster_list = conn.get(cls.rest_entity_path, params=params)
```

Comment:
afaik clusters v1 does not support pagination

Reply:
Yes that's right. When using v1.2, it just returns all the clusters. Should I modify it to log a warning that v1.2 doesn't support pagination and will list all the clusters?